docs/static/releasenotes.asciidoc

```diff
@@ -64,6 +65,21 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
 
+[[logstash-7-17-5]]
+=== Logstash 7.17.5 Release Notes
+
+[[notable-7-17-5]]
+==== Notable issues fixed
+* Fix a possible corruption of Persistent Queue during a crash of the Logstash process https://github.com/elastic/logstash/pull/14147[#14147]
+* Fixes to the pre-flight check for starting a pipeline with the Persistent Queue:
+** we correctly validate free capacity on the _volume_ on which the pipeline's PQ is configured https://github.com/elastic/logstash/pull/14212[#14212]
+** starting with insufficient free disk space for the configured PQ size is now a warning instead of an error https://github.com/elastic/logstash/pull/14267[#14267]
```

Comment:
```suggestion
** We correctly validate free capacity on the _volume_ on which the pipeline's PQ is configured https://github.com/elastic/logstash/pull/14212[#14212]
** Starting with insufficient free disk space for the configured PQ size is now a warning instead of an error https://github.com/elastic/logstash/pull/14267[#14267]
```